client/src/components/ProductList/ProductList.js

```diff
@@ -14,10 +14,11 @@ export default function ProductList(props) {
   function minPrice(shops) {
     const regularPrice = shops
       .map(product => product.regularPrice)
-      .sort((a, b) => a > b)
+      .sort((a, b) => a < b)
     const salesPrice = shops
       .map(product => product.salesPrice)
       .sort((a, b) => a > b)
+      .filter(Boolean)
```

Comment:
Just out of curiosity: what does this filter do exactly?

Reply:
This clean's up all empty Array Elements ✨